src/vm/amd64/unixasmhelpers.S

```diff
@@ -172,6 +172,98 @@ NESTED_ENTRY JIT_RareDisableHelper, _TEXT, NoHandler
 
 NESTED_END JIT_RareDisableHelper, _TEXT
 
+#ifdef FEATURE_HIJACK
+
+//------------------------------------------------
+// OnHijackScalarTripThread
+//
+NESTED_ENTRY OnHijackScalarTripThread, _TEXT, NoHandler
```

Comment:
Spoke to @janvorli about this. We'll wait on doing this until after the work for passing structs in registers is ready to be merged. Once we do, we'll also need to ensure that object refs in the registers are handled properly.